amps/ags/rm-community/rm-community-repo/pom.xml

```diff
@@ -84,6 +84,12 @@
          <artifactId>junit</artifactId>
          <scope>test</scope>
       </dependency>
+       <dependency>
+           <groupId>com.epam.reportportal</groupId>
+           <artifactId>agent-java-testng</artifactId>
+           <version>5.4.0</version>
```

Comment:
Can't we set the `agent-java-testng` and `okhttp` versions as version variables? This applies to all declarations in the code.